gradle/verification-metadata.xml

```diff
@@ -1347,6 +1347,11 @@
             </sha256>
          </artifact>
       </component>
+      <component group="org.jetbrains.kotlin" name="kotlin-gradle-plugin" version="1.7.0-dev-1904">
```

Comment:
Please remove this entry from the verification metadata. The artifact is signed, right?